src/BDoc.cpp

```diff
@@ -490,6 +490,10 @@ void BDoc::parseManifestAndLoadFiles(const ZipSerialize &z, const vector<string>
     {
         THROW("Failed to parse manifest XML: %s (xsd path: %s)", e.what(), Conf::instance()->xsdPath().c_str());
     }
+    catch (...)
```

Comment:
Nothing to catch.
Exception is thrown by:
   c:\build\xerces\src\xercesc\internal\IGXMLScanner.cpp
...
    catch(const OutOfMemoryException&)
    {
        // This is a special case for out-of-memory
        // conditions, because resetting the ReaderMgr
        // can be problematic.
        resetReaderMgr.release();

        throw;
    }